features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/App.jsx

```diff
@@ -50,6 +52,31 @@ class Store extends React.Component {
     constructor(props) {
         super(props);
         LoadableProtectedApp.preload();
+        this.state = {
+            settings: null,
+        };
+        this.retrieveSettings = this.retrieveSettings.bind(this);
+    }
+
+    /**
+     *  Mounting
+     */
+    componentDidMount() {
+        this.retrieveSettings();
```

Comment:
Why can't we put the body of `retrieveSettings` method into `componentDidMount`, `retrieveSettings` haven't used in anywhere else ?

Reply:
no, i'll remove retrieveSettings method and then do the logic inside componentDidMount.